stat.c

```diff
@@ -414,6 +414,18 @@ static void display_lat(const char *name, unsigned long long min,
 	free(maxp);
 }
 
+static double convert_agg_kbytes_percent(struct group_run_stats *rs, int ddir, int mean)
```

Comment:
Nice! One question is should you just take agg and expect the caller to do pass you the value rather than a pointer that you have to deference?

Reply:
I prefer the pointer here to be dereferenced since I prefer providing meaningful insight to the function since this aggregation is specific to this run_stats or am I missing something here